pkg/controllers/interruption/metrics.go

```diff
@@ -35,15 +35,15 @@ var (
 			Help:      "Count of messages received from the SQS queue. Broken down by message type and whether the message was actionable.",
 		},
 		[]string{messageTypeLabel},
-	)
+	) //stability=STABLE
```

Comment:
```suggestion
	) // +karpenter:stability=stable
```

nit: Can we do something that looks similar to the [`kubebuilder` markers](https://book.kubebuilder.io/reference/markers)